Src/AutoNSubstitute/AutoNSubstitute.csproj

```diff
@@ -64,6 +64,7 @@
   <ItemGroup>
     <Compile Include="AutoConfiguredNSubstituteCustomization.cs" />
     <Compile Include="AutoNSubstituteCustomization.cs" />
+    <Compile Include="AutoSubstituteCustomization.cs" />

```

Comment:
> I don't understand why renaming the new SubstituteXxx classes to include the N prefix would be a breaking change

Perhaps I miswrote somewhere. Renaming the *new* types in this pull request would *not* be a breaking change.

What I meant to state was that renaming *existing* types would be a breaking change. Sorry about the confusion.